docs/sources/enterprise/license/license-restrictions.md

```diff
@@ -12,135 +12,41 @@ When you become a Grafana Enterprise customer, you receive a license that govern
-
-> When you delete a user you also delete their preferences, dashboard and folder permissions, and references to that user from the dashboard versions they have updated.
-
-Users who are synced via SAML, Oauth, or LDAP will also lose these attributes.
+1. Review the utilization count on the **Utilization** panel.
 
-### Update user permissions for dashboards
+## Tiered licensing (deprecated)
 
-You can change the access a user has to specific dashboards. For billing purposes, Grafana charges for the highest level permission assigned to the user. For example, if you assign editor/admin dashboard permissions to a viewer user, Grafana counts that user as an editor/admin.
-
-To change user permissions for dashboards:
-
-1. Sign in to Grafana as a system administrator.
-
-1. Hover your mouse over **Server Admin** (the shield icon) and click **Statistics and licensing**.
-
-1. In the **Dashboard and Folder Permissions** area, click the dashboard in the \*\*Resource title (URL) column.
-
-   The **Permission** page opens.
-
-1. Use the dropdown list to select a role.
-
-The next time the user signs in, the **Utilization** panel on the **Statistics and licensing** page displays updated data.
-
-## Combined licensing
-
-As of Grafana Enterprise Version 8.3, you can purchase (or transition to) combined licensing. With combined licensing, you purchase a specific number of users, and you are free to distribute those users across all roles, in any combination.
-For example, if you purchase 150 licenses, you can have 20 admins, 70 editors, and 60 viewers, or you can have 10 admins, 100 editors, and 40 viewers. This change reduces license complexity.
-
-### Transition to combined license model
-
-To transition from the tiered licensing model to the combined license model, contact your Grafana account team and request to switch to combined user pricing. Once you update your contract with the account team, they will issue you a new license token.
-For instructions about how to update your license, refer to [Activate an Enterprise license]({{< relref "./activate-license.md" >}}).
-
-After you apply the token, Grafana Enterprise resets your license and updates the user counts on the **Utilization** panel.
-
-> If you are running Grafana Enterprise 8.2 or earlier, the license grants you the total number of licensed users _for each user type_.
-
-For example, if your current license includes 60 viewers and 40 editor/admins, the new license includes 100 viewers and 100 editor/admins. Grafana Enterprise 8.3 removes the distinction between viewers and editor/admins as shown on the **Utilization** panel.
-
-Before you upgrade to Grafana 8.3, ensure that the total number of active users in Grafana does not exceed the number of users in your combined license. If it does, then new users cannot sign in to Grafana 8.3 until the active user count returns below the licensed limit.
-
-## Determining your current license type
```

Comment:
This section seems more relevant with the deprecation, so users can determine whether they need to update their licensing, and IMO should not be removed.

Reply:
Yes you're right, it's still relevant for users coming from before 8.3 to 9.0, I put it back in the doc.

Reply:
I'm sorry, I should have been clearer. The "transition to combined licensing model" section still appears to be relevant. Otherwise, I'm not sure how 9.0 users with tiered licensing will be able to transition to combined licensing.